modules/get-started/pages/licensing/overview.adoc

```diff
@@ -0,0 +1,88 @@
+= Redpanda Licenses, Editions, and Enterprise Features
+:description: Redpanda Self-Managed is free and source-available at the Redpanda GitHub repo. Additional features are included with an Enterprise license.
```

Comment:
This page description (and therefore the index page) suggested to me that only licensing etc for the "core" Redpanda product would be included, but the introduction mentions also the the Redpanda Console and Redpanda Connect. Would it be worth updating?

Reply:
```suggestion
:description: Learn about Redpanda licensing for Self-Managed, Console, and Connect, available in both Community and Enterprise editions. Understand licensing requirements and how to access enterprise features with a valid license key.
```

Reply:
This description gets used as the meta description for search engines. It can't include links.